pontoon/base/models/project_locale.py

```diff
@@ -29,11 +29,20 @@ def visible(self):
             project__system_project=False,
         ).distinct()
 
-    def stats_data(self, project):
-        query = self.filter(
-            locale__translatedresources__resource__project=project
-        ).prefetch_related("locale")
-        tr = "locale__translatedresources"
+    def stats_data(self, project=None, locale=None):
+        if project:
+            query = self.filter(
+                locale__translatedresources__resource__project=project
+            ).prefetch_related("locale")
+            tr = "locale__translatedresources"
+        elif locale:
+            query = self.filter(
+                project__resources__translatedresources__locale=locale,
+                project__resources__translatedresources__resource__project__disabled=False,
+                project__resources__translatedresources__resource__project__system_project=False,
+                project__resources__translatedresources__resource__project__visibility="public",
```

Comment:
Is that not working OK?

```suggestion
                project__disabled=False,
                project__system_project=False,
                project__visibility="public",
```